GitUI/CommandsDialogs/RevisionFileTreeControl.cs

```diff
@@ -51,26 +51,29 @@ public partial class RevisionFileTreeControl : GitModuleControl
         private readonly Stack<string> _lastSelectedNodes = new();
         private readonly IRevisionFileTreeController _revisionFileTreeController;
         private readonly IFullPathResolver _fullPathResolver;
-        private readonly IFindFilePredicateProvider _findFilePredicateProvider;
+        private readonly IFindFilePredicateProvider _findFilePredicateProvider = new FindFilePredicateProvider();
```

Comment:
Why is this necessary?

Reply:
It is not necessary for this PR, it was a change I had done (and believe should be changed) that was refactored to a separate commit